StudioCore/ParamEditor/ParamBank.cs

```diff
@@ -161,8 +161,6 @@ private static void LoadParamFromBinder(IBinder parambnd, ref Dictionary<string,
                 }
                 if (p.ApplyParamdefCarefully(_paramdefs[p.ParamType]))
                     paramBank.Add(Path.GetFileNameWithoutExtension(f.Name), p);
```

Comment:
Add a warning to the warning list if ApplyParamDefCarefully fails

Reply:
I did have this temporarily. I don't know why I decided the way I did, but we are also ignoring several other params.